extensions/cornerstone/src/ConnectedCornerstoneViewport.js

```diff
@@ -23,36 +23,41 @@ const MEASUREMENT_ACTION_MAP = {
 const mapStateToProps = (state, ownProps) => {
   let dataFromStore;
 
+  // TODO: This may not be updated anymore :thinking:
```

Comment:
👀

Reply:
Changing state/props to be cleaner is the rabbit hole of the larger PR. On my list, but just need this fixes merged first.